### DISCUSSION TITLE
Hope we could set ConstLabels for go_collector

### DISCUSSION THREAD

**Discussion by LeGamerDc** [prometheus/client_golang#1078] [General]
we need to tags our metrics to tell apart from other process.

**Comment by beorn7** [comment]
https://pkg.go.dev/github.com/prometheus/client_golang@v1.12.1/prometheus#WrapRegistererWith should help you with that.

Note, however, that you should use [target labels](https://www.robustperception.io/target-labels-not-metric-name-prefixes/) to tell apart metrics from different processes.

**Reply by LeGamerDc** [reply]
thank you, that's exactly what we need!!!
